gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -188,6 +188,9 @@
 	public static final String ICON_SAKAI = "icon-sakai--";
 	public static final String ALL = "all";
 
+	private static final String SAK_PROP_ALLOW_STUDENTS_TO_COMPARE_GRADES = "gradebookng.allowStudentsToCompareGradesWithClassmates";
+	private static final Boolean SAK_PROP_ALLOW_STUDENTS_TO_COMPARE_GRADES_DEFAULT = false;
```

Comment:
Boolean.FALSE;

Reply:
I don't see why `Boolean.FALSE` is necessary here.